### PR TITLE
[Cognitive Services] add new regions: Canada Central, Central Inida, Japan East and UK South

### PR DESCRIPTION
**Pull request by yangyuan**
## Description
https://github.com/Azure/azure-rest-api-specs/pull/3308
Add new regions: Canada Central, Central Inida, Japan East and UK South

There are two commits:
1. AutoGenerate codes (also upgrade one generate.cmd to generate.ps1).
2. Update `*.csproj` and `AssemblyInfo.cs` versions.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Language Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming Microsoft Azure Cognitive Services Language APIs.")]
 
-[assembly: AssemblyVersion("1.0.1.0")]
-[assembly: AssemblyFileVersion("1.0.1.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Properties/AssemblyInfo.cs]: AssemblyVersion stays the same, AssemblyFileVersion needs to be updated

File: src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Language Client Library")]
 [assembly: AssemblyDescription("Provides API functions for consuming Microsoft Azure Cognitive Services Language APIs.")]
 
-[assembly: AssemblyVersion("1.0.1.0")]
-[assembly: AssemblyFileVersion("1.0.1.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
+[assembly: AssemblyFileVersion("1.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Language/TextAnalytics/TextAnalytics/Properties/AssemblyInfo.cs]: AssemblyVersion stays the same

File: src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj [hunk 3]
```diff
@@ -5,7 +5,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.ContentModerator</PackageId>
     <Description>This client library provides access to the Microsoft Cognitive Services Content Moderator APIs.</Description>
-    <Version>0.12.0-preview</Version>
+    <Version>0.12.1-preview</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.ContentModerator</AssemblyName>
     <PackageTags>ContentModerator;Content Moderator;</PackageTags>
     <PackageReleaseNotes>This is a preview release of the Cognitive Services Content Moderator SDK.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ContentModerator/ContentModerator/Microsoft.Azure.CognitiveServices.ContentModerator.csproj]: Please update `PackageReleaseNotes`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great other than a few comments.

**Comment by dsgouda** [comment]
@yangyuan Please make sure the `AssemblyVersion` stays same but `AssemblyFileVersion` changes

File: src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (yangyuan) [reply src/SDKs/CognitiveServices/dataPlane/Language/Language/Language/Properties/AssemblyInfo.cs]: Just want to confirm, AssemblyVersion only changes on breaking changes?

**Comment by yangyuan** [comment]
Hi @dsgouda
Just want to confirm, AssemblyVersion only changes on breaking changes?

**Comment by dsgouda** [comment]
`AssemblyVersion` changes only when major version is bumped. 
There can be breaking changes without a major version bump in preview however

**Comment by dsgouda** [comment]
@yangyuan Please fix failing test cases too

**Comment by yangyuan** [comment]
Was a syntax mistake in csproj, should be fixed now.

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs [hunk 4]
```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Cognitive Services ComputerVision SDK")]
 [assembly: AssemblyDescription("Provides access to the Microsoft Cognitive Services ComputerVision APIs.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs]: The `AssemblyVersion` stays the same, `AssemblyFileVersion` changes

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great apart from the comment
Please ensure all the assemblyinfo.cs files are versioned appropriately

File: src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs (hunk 4 above)
Comment (yangyuan) [reply src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Properties/AssemblyInfo.cs]: Ahh sorry, missed this.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**